swift/Podfile

```diff
@@ -4,6 +4,7 @@ target 'TrustWalletCore' do
   use_frameworks!
 
   pod 'SwiftProtobuf', '~> 1.2.0'
+  pod 'BigInt'
```

Comment:
Noo

Reply:
It’s temporary to demonstrate the bug, will remove it